sequencer/src/bin/submit-transactions.rs

```diff
@@ -0,0 +1,161 @@
+//! Utility program to submit random transactions to an Espresso Sequencer.
+
+use async_compatibility_layer::logging::{setup_backtrace, setup_logging};
+use async_std::task::{sleep, spawn};
+use bytesize::ByteSize;
+use clap::Parser;
+use commit::{Commitment, Committable};
+use derive_more::From;
+use futures::{
+    channel::mpsc::{self, Sender},
+    sink::SinkExt,
+    stream::StreamExt,
+};
+use hotshot_query_service::{
+    availability::{BlockQueryData, QueryablePayload},
+    Error,
+};
+use rand::{thread_rng, Rng, RngCore};
+use sequencer::{options::parse_duration, SeqTypes, Transaction};
+use snafu::Snafu;
+use std::{collections::HashSet, time::Duration};
+use surf_disco::{Client, Url};
+
+/// Submit random transactions to an Espresso Sequencer.
+#[derive(Clone, Debug, Parser)]
+struct Options {
+    /// Minimum size of transaction to submit.
+    #[clap(long, default_value = "1", value_parser = parse_size)]
+    min_size: usize,
+
+    /// Maximum size of transaction to submit.
+    #[clap(long, default_value = "1kb", value_parser = parse_size)]
+    max_size: usize,
+
+    /// Optional delay between submitting transactions.
+    ///
+    /// Can be used to moderate the rate of submission.
+    #[clap(long, value_parser = parse_duration)]
+    delay: Option<Duration>,
+
+    /// Maximum number of unprocessed transaction submissions.
+    ///
+    /// This can be used to apply backpressure so that the tasks submitting transactions do not get
+    /// too far ahead of the task processing results.
+    #[clap(long, default_value = "1000")]
+    channel_bound: usize,
+
+    /// Number of parallel tasks to run.
+    #[clap(short, long, default_value = "1")]
+    jobs: usize,
+
+    /// URL of the query service.
```

Comment:
This doesn't have the clap derive. What does that do?

Reply:
Makes it a (required) positional argument